### PR TITLE
Apply JS gov

### PR DESCRIPTION
**Pull request by achamayou**
Apply actions from JS governance.

### CODE REVIEW

File: src/js/wrap.h [hunk 1]
```diff
@@ -22,14 +22,27 @@ namespace js
   extern JSClassDef kv_map_handle_class_def;
   extern JSClassDef body_class_def;
 
+  enum TxAccess
```
Comment (letmaik) [inline src/js/wrap.h:25]: enum class?

File: src/runtime_config/constitution.js [hunk 2]
```diff
@@ -104,8 +88,18 @@ const actions = new Map([
...
+    ),
+  ],
+  [
+    "remove_user",
+    new Action(
+      function (args) {
+        return typeof args.user_id === "string";
+      },
+      function (args) {
+        ccf.kv["public:ccf.gov.users.certs"].delete(args.user_id);
```
Comment (letmaik) [inline src/runtime_config/constitution.js:101]: Needs strToBuf

Comment (achamayou) [reply src/runtime_config/constitution.js]: Yes, thank you.

File: src/js/wrap.h (hunk 1 above)
Comment (achamayou) [reply src/js/wrap.h:25]: Sorry?

**Review by letmaik — APPROVED** [review]

File: src/runtime_config/constitution.js [hunk 3]
```diff
@@ -104,8 +88,19 @@ const actions = new Map([
...
-      function (args, tx) {
-        return true;
+      function (args) {}
+    ),
+  ],
+  [
+    "remove_user",
+    new Action(
+      function (args) {
+        return typeof args.user_id === "string";
```
Comment (jumaffre) [inline src/runtime_config/constitution.js:98]: We should probably add a function to check that a giver id is valid (i.e. hex-encoded string of SHA-256).

**Review by jumaffre — APPROVED** [review]

File: src/runtime_config/constitution.js (hunk 3 above)
Comment (achamayou) [reply src/runtime_config/constitution.js:98]: Yes, let's refine the actions as we implement them, this is just a testcase for the apply path, not necessarily the final remove_user action.

**Pull request merged**